go/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Move the go directory into $PREFIX/ so it can be built in its install location.
+cd .. && mv go $PREFIX/ && cd $PREFIX/go
```

Comment:
This is not the way conda-build works. Doesn't Go has a `make install` step or something?